package.json

```diff
@@ -68,6 +68,6 @@
     "keytar": "^7.6.0"
   },
   "engines": {
-    "node": ">=10.12.0"
+    "node": ">=14.0.0"
```

Comment:
Should we mark these PR's as breaking changes to bump the version?
Because of the change ^